owl/SOMA-SAY.owl

```diff
@@ -139,34 +101,34 @@ ObjectPropertyRange(SOMA:movesObject <http://www.ontologydesignpatterns.org/ont/
 #   Classes
 ############################
 
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#AssertionTask> (Assertion task)
+# Class: SOMA:AssertionTask (Assertion task)
 
 SubClassOf(SOMA:AssertionTask ObjectAllValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies> ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasParticipant> SOMA:DeclarativeClause)))
 
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#CausedMotionTheory> (Caused motion theory)
+# Class: SOMA:CausedMotionTheory (Caused motion theory)
 
 AnnotationAssertion(rdfs:comment SOMA:CausedMotionTheory "A schematic theory describing a situation in which an agent performs an action which causes an object to move along a certain path. A popular example being \"He sneezed the napkin off the table.\" (Goldberg 1995)
 
 Let xA, xP be objects filling the agent, patient roles of this schema. Then one can infer that xA movesObject xP.")
 AnnotationAssertion(rdfs:label SOMA:CausedMotionTheory "Caused motion theory"@en)
 SubClassOf(SOMA:CausedMotionTheory SOMA:ImageSchemaTheory)
-SubClassOf(SOMA:CausedMotionTheory ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines> SOMA:AgentRole))
```

Comment:
Was this moved somewhere else or removed? Why?

Reply:
The AgentRole has been renamed to PerformerRole, becaue I wanted it to be clearer for the Capabilities.
I can undo that of course, if wanted.
(The axiom that seems missing is still there, but with PerformerRole)